src/app/js/admin/Appbar/JobProgress.js

```diff
@@ -109,6 +117,18 @@ const JobProgressComponent = props => {
                             </Typography>
                         )}
                     </div>
+
+                    {progress?.type === 'publisher' && (
+                        <Button
```

Comment:
I don't see a pointer cursor on the video, is it only the video or does this button miss some styles?